### PR TITLE
Update deps and header design 

### PR DESCRIPTION
**Pull request by benja-M-1**
Hi @loopDelicious :wave:

In this PR I have
- fixed the makefile: some commands were referencing build-extension but the name was still build
- configured webpack so that it does not open a browser on start
- updated some dependencies
- fixed some typescript errors
- made the header to look like the Docker's recommanded one
- added an alert to inform the user they where they can find their PMKA or use the public one so that when clicking on the link it is opened in a browser and not in Docker Desktop.

Screenshot 2022-08-25 at 12 34 31

### CODE REVIEW

File: client/src/App.tsx [hunk 1]
```diff
@@ -131,27 +131,16 @@ function App() {
...
                   id="apikey-input"
                   label={[
-                    "Enter a ",
-                    <a
-                      href="https://go.postman.co/settings/me/api-keys"
-                      style={{ color: "rgb(25, 118, 210)" }}
-                    >
-                      Postman API key
-                    </a>,
-                    " to retrieve your collections.",
```
Comment (benja-M-1) [inline client/src/App.tsx:154]: Damn, this should not be part of this PR, I am going to fix that.

**Comment by loopDelicious** [comment]
thanks @benja-M-1
